README.md

```diff
@@ -1500,19 +1502,17 @@ extension DataRequest {
             guard error == nil else { return .failure(BackendError.network(error: error!)) }
 
             let jsonSerializer = DataRequest.jsonResponseSerializer(options: .allowFragments)
-            let result = jsonSerializer.serializeResponse(request, response, data, error)
+            let result = jsonSerializer.serializeResponse(request, response, data, nil)
```

Comment:
Added in [0a05e2b](https://github.com/Alamofire/Alamofire/commit/880dc772a12d1ec490eb8aee029115ec8).